updater/lib/dependabot/updater/group_update_creation.rb

```diff
@@ -19,24 +19,12 @@ module Dependabot
   class Updater
     extend T::Sig
 
-    module GroupUpdateCreation
+    class GroupUpdateCreation < OperationBase
```

Comment:
I am not sure this is the best approach. This is a mixin in Ruby. I had originally attempted to type this in #8319, but now that I have more experience with Ruby, I realise this isn't the correct approach. Check out #9917 for an example of how I typed a helper mixin in `pub`.

Reply:
Actually, reviewing further down this PR, it looks like you've just added this as part of the class hierarchy. I believe that will work as well!

Reply:
That approach gets us to `#typed: true`, but once you do `#typed: strict` Sorbet complains that you never populated those attr_reader's.